cnf-tests/testsuites/e2esuite/dpdk/numa_node_sriov.go

```diff
@@ -0,0 +1,163 @@
+		Expect(err).ToNot(HaveOccurred())
+		if isSNO {
+			Skip("Single Node openshift not yet supported")
+		}
+
+		perfProfile, err := performanceprofile.FindDefaultPerformanceProfile(performanceProfileName)
+		Expect(err).ToNot(HaveOccurred())
+		if !performanceprofile.IsSingleNUMANode(perfProfile) {
+			Skip("SR-IOV NUMA test suite expects a performance profile with 'single-numa-node' to be present")
+		}
+
+		err = namespaces.Create(sriovnamespaces.Test, client.Client)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Clean SRIOV policies and networks")
+		networks.CleanSriov(sriovclient)
+
+		By("Discover SRIOV devices")
+		sriovCapableNodes, err := sriovcluster.DiscoverSriov(sriovclient, namespaces.SRIOVOperator)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(len(sriovCapableNodes.Nodes)).To(BeNumerically(">", 0))
+		testingNode, err := nodes.GetByName(sriovCapableNodes.Nodes[0])
+		Expect(err).ToNot(HaveOccurred())
+		By("Using node " + testingNode.Name)
+
+		sriovDevices, err := sriovCapableNodes.FindSriovDevices(testingNode.Name)
+		Expect(err).ToNot(HaveOccurred())
+
+		numa0Device, err := findDeviceOnNUMANode(testingNode, sriovDevices, "0")
+		Expect(err).ToNot(HaveOccurred())
+		By("Using NUMA0 device " + numa0Device.Name)
+
+		numa1Device, err := findDeviceOnNUMANode(testingNode, sriovDevices, "1")
+		Expect(err).ToNot(HaveOccurred())
+		By("Using NUMA1 device " + numa1Device.Name)
+
+		// SriovNetworkNodePolicy
+		// NUMA node0 device excludeTopology = true
+		// NUMA node0 device excludeTopology = false
+		// NUMA node1 device excludeTopology = true
+		// NUMA node1 device excludeTopology = false
+
+		By("Create SRIOV policies and networks")
+
+		ipam := `{ "type": "host-local", "subnet": "192.0.2.0/24" }`
+
+		createSriovNetworkAndPolicyForNumaAffinityTest(8, numa0Device, "#0-3",
+			"test-numa-0-exclude-topology-false-", testingNode.Name,
+			"testNuma0ExcludeTopoplogyFalse", ipam, false)
+
+		createSriovNetworkAndPolicyForNumaAffinityTest(8, numa0Device, "#4-7",
+			"test-numa-0-exclude-topology-true-", testingNode.Name,
+			"testNuma0ExcludeTopoplogyTrue", ipam, true)
+
+		createSriovNetworkAndPolicyForNumaAffinityTest(8, numa1Device, "#0-3",
+			"test-numa-1-exclude-topology-true-", testingNode.Name,
+			"testNuma1ExcludeTopoplogyFalse", ipam, false)
+
+		createSriovNetworkAndPolicyForNumaAffinityTest(8, numa1Device, "#4-7",
+			"test-numa-1-exclude-topology-true-", testingNode.Name,
+			"testNuma1ExcludeTopoplogyTrue", ipam, true)
+
+		By("Waiting for SRIOV devices to get configured")
+		networks.WaitStable(sriovclient)
+	})
+
+	BeforeEach(func() {
+		By("Clean any pods in " + sriovnamespaces.Test + " namespace")
+		namespaces.CleanPods(sriovnamespaces.Test, sriovclient)
+	})
+
+	It("Validate the creation of a pod with excludeTopology set to False and an SRIOV interface in a different NUMA node than the pod", func() {
+		pod := pods.DefinePod(sriovnamespaces.Test)
+		pods.RedefineWithGuaranteedQoS(pod, "1", "100m")
```

Comment:
nit: can you use the same pod = ... pattern here? Or, is there a reason why it was not done?